Update Exploring Datasets.ipynb

IqraMahmood opened the pull request:
The following headings are added for better exploration of a given dataset: 
- Python Basic operations
- Finding the Pandas version
- Import dataset
- See the top5 entries
- See the last 5 entries
- Number of columns in the dataset
- Number of observations in the dataset
- Name of all the columns
- Checking the type of each column
- select a column by dtype
- Summarize the data frames
 - Value Counts
- Finding unique values in a dataset
- Data Filtering
- Sorting
 **Also, make some formatting in the heading like making the important heading bold.**

IqraMahmood committed:
Update Exploring Datasets.ipynb

The following headings are added for better exploration of a given dataset:
Python Basic operations
Finding Pandas version
Import dataset
See the top5 entries
See the last 5 entries
Number of columns in the dataset
Number of observations in the dataset
Name of all the columns
Checking the type of each column
select a column by dtype
Summarize the data frames
Value Counts
Finding unique values in a dataset
Data Filtering
Sorting
 Also, make some formatting in the heading like making the important heading bold.

MartinSeeler commented:
Hi @IqraMahmood , thanks again for your contribution. This looks better than the last one 👍 

To merge this, I'd like to ask you for some small changes:

1. [ ] Change `See the first 10 entries` to `See the first 5 entries`, because you only show 5
2. [ ] Add an exercise to `Exploring Datasets Yourself.ipynb` for `Select a column by dtypes
`, because that is a new topic you covered and should have an exercise for the reader afterwards
3. [ ] Add an exercise to `Exploring Datasets Yourself.ipynb` for `Filtering by large categories`, because that is a new topic you covered and should have an exercise for the reader afterwards
4. [ ] Add an exercise to `Exploring Datasets Yourself.ipynb` for `Sorting`, because that is a new topic you covered and should have an exercise for the reader afterwards

After that, I will merge it 🚀

IqraMahmood committed:
Update heading in `Exploring Datasets.ipynb`

Change `See the first 10 entries` to `See the first 5 entries`

IqraMahmood committed:
Add exercises to Exploring Datasets Yourself.ipynb

- Add an exercise to Exploring Datasets Yourself.ipynb for Select a column by dtypes 
- Add an exercise to Exploring Datasets Yourself.ipynb for Filtering by large categories
- Add an exercise to Exploring Datasets Yourself.ipynb for Sorting
